extensions/pythonprocessors/google/SentimentAnalyzer.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+  Install the following with pip ( or pip3 )
+  
+  pip install google-cloud-language
```

Comment:
Import can be added to try-cache blocks, so error messages can be printed to warn the user to install.

Reply:
I actually had it catch and attempt install...that didn't work in the confines of where it was executing. Is that something that's generally ill advised?

A warning should be printed that there is a missing dependency. This won't prevent startup, but will be in the log file.